### PR TITLE
Fixes timestamp overlay bug

### LINKED ISSUE #1067: Timestamp appears ontop of message bubble
I only notice this when the chat controller gets updated with multiple messages. For example, coming in and out of network and you have queued messages to be displayed. Happens on the update. 

photo

### PR DESCRIPTION
**Pull request by jpmcglone**
resolve #1067

### CODE REVIEW

File: Code/Controllers/ATLConversationViewController.m [hunk 1]
```diff
@@ -483,9 +483,14 @@ - (void)configureHeader:(ATLConversationCollectionViewHeader *)header atIndexPat
...
     if ([self shouldDisplayDateLabelForSection:indexPath.section]) {
         [header updateWithAttributedStringForDate:[self attributedStringForMessageDate:message]];
+    } else {
+        [header updateWithAttributedStringForDate:[[NSAttributedString alloc] initWithString:@"" attributes:nil]];
     }
+    
     if ([self shouldDisplaySenderLabelForSection:indexPath.section]) {
         [header updateWithParticipantName:[self participantNameForMessage:message]];
+    } else {
+        [header updateWithAttributedStringForDate:[[NSAttributedString alloc] initWithString:@"" attributes:nil]];
```
Comment (prabhupb) [inline Code/Controllers/ATLConversationViewController.m]: Shouldn't this be ` [header updateWithParticipantName:@""];` ?

Comment (prabhupb) [inline Code/Controllers/ATLConversationViewController.m]: Currently this hides all the timestamps if the sender label is not displayed.

Comment (edwinruan) [inline Code/Controllers/ATLConversationViewController.m]: Is that a way to display timestamp but hide sender Label? Look like hiding both is just a workaround.

Comment (edwinruan) [inline Code/Controllers/ATLConversationViewController.m]: Is that a way to display timestamp but hide sender Label? Look like hiding both is just a workaround.

**Comment by vinzen** [comment]
#1404

File: Code/Controllers/ATLConversationViewController.m (hunk 1 above)
Comment (vinzen) [inline Code/Controllers/ATLConversationViewController.m]: PR created #1404

**Pull request merged**